test22.js

```diff
@@ -0,0 +1,5 @@
+var foo = '' ;
+var poco = '';
+if(foo  === 'bar') {
+  console.log( 'hey' )
```

Comment:
Codacy has a fix for the issue: [Replace `·'hey'·)` with `"hey");`](https://release.k8s.dev.codacy.org/gh/machadoit-test-org/eslint-example/pullRequest?prid=5795)
```suggestion
  console.log("hey");
```